python/ngen_cal/src/ngen/cal/objectives.py

```diff
@@ -19,6 +19,13 @@ def normalized_nash_sutcliffe(observed, simulated):
     nse = nash_sutcliffe(observed, simulated)
     return 1/(2-nse)
 
+def inverted_nnse(observed, simulated):
```

Comment:
This doesn't appear to ever get used... though it probably should be an option up in strategy.py if we want to use NSE with PSO, I'm guessing? Same discussion as with KGE there, and we should harmonize the names if that makes sense...though the result of 1-nnse isn't quite the same as 1-kge, range-wise, so it may be better to call them different things.

Reply:
Any of these objective functions can be used by setting the objective value in the config to the python module path, so it is a viable option.  As far as naming goes, happy to work out better conventions.  Can we open an issue for that and address them after this merge?